packages/data-table-server/src/dataTableService/services/AnalyticsDataTableService.ts

```diff
@@ -16,8 +16,8 @@ const requiredParamsSchema = yup.object().shape({
   hierarchy: yup.string().default('explore'),
   dataElementCodes: yup.array().of(yup.string().required()).required(),
   organisationUnitCodes: yup.array().of(yup.string().required()).required(),
-  startDate: yup.date().default(new Date(EARLIEST_DATA_DATE_STRING)),
-  endDate: yup.date().default(() => new Date()),
+  startDate: yup.date(),
+  endDate: yup.date(),
```

Comment:
```suggestion
  startDate: yup.date().default(getDefaultStartDate),
  endDate: yup.date().default(getDefaultEndDate),
```
With functions somewhere:
```
const getDefaultStartDate = () => {
  return new Date(convertPeriodStringToDateRange(getDefaultPeriod())[0]);
};

const getDefaultEndDate = () => {
  return new Date(convertPeriodStringToDateRange(getDefaultPeriod())[1]);
};
```

Thoughts on this? I feel the main plus is that we keep the 'default value' in the parameter definition which can be useful when populating defaults in the viz-builder

Reply:
Ah cool sounds good. I've put defaultValue back now.